2023-12-02 16:08:43,126 - test_dem.py[line:39] - ERROR: 断言失败，用例为：['老黄1号店', 'laohuang1hao', '老黄1号店的维度']。期望结果为:添加维度成功 - root
2023-12-02 16:08:46,742 - test_dem.py[line:39] - ERROR: 断言失败，用例为：['', 'laohuang1hao', '老黄1号店的维度']。期望结果为:添加维度失败 - root
2023-12-02 16:08:51,434 - test_dem.py[line:39] - ERROR: 断言失败，用例为：['老黄1号店', '', '老黄1号店的维度']。期望结果为:添加维度失败 - root
